omd.opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Philippe Wang <philippe.wang@gmail.com>"
+authors: [ "Philippe Wang <philippe.wang@gmail.com>" ]
+version: "1.3.1"
+authors: [ "Philippe Wang <philippe.wang@gmail.com>" ]
+license: "ISC"
+homepage: "https://github.com/ocaml/omd"
+dev-repo: "https://github.com/ocaml/omd.git"
+bug-reports: "https://github.com/ocaml/omd/issues"
+tags: [ "org:ocamllabs" "org:mirage"  ]
+depends: [
+  "base-bytes"
+  "base-bigarray"
+  "dune" {build & >= "1.0"}
```

Comment:
L’attributo lang 1.1 usato in dune-project è compatibile con dune 1.0?

Reply:
Thanks, fixed.